xml/depl_conf_admin_crowbar.xml

```diff
@@ -631,7 +631,7 @@
   template file <filename>/etc/crowbar/network.json</filename>. This section
   explains the file in detail. Settings in this file are applied to all
   nodes in &cloud;. (See <xref
-   linkend="sec.network.json.resolve"/> to learn how to verify your correct network interface names.)
+   linkend="sec.network.json.resolve"/> to learn how to verify your correctF network interface names.)
```

Comment:
also, "verify your correct names" seems like a pleonasm. => "verify the network interface names."